tests/token/erc721/test_ERC721SafeMintableMock.py

```diff
@@ -1,19 +1,16 @@
 import pytest
```

Comment:
Just curious about why do we have a test for a Mock.

Reply:
Good question! The main motivation was to include tests for `safeMint` since it's not included in the presets thus far